### PR TITLE
Design and content tweaks before launch

### LINKED ISSUE #167: Design and content tweaks
1. Stop external link icons breaking to new line, and adjust vertical positioning
Screen Shot 2023-01-16 at 11 02 21 AM
2. Change to generic cover
screen-shot-2023-01-17-at-12 55 03-pm
3. Unify paragraph font size in Uses Cases section
4. Update CTA from 'Try WordPress' to 'Get WordPress'

### PR DESCRIPTION
**Pull request by adamwoodnz**
Closes #167, see description for changes.

### Screenshots

| Desktop | Tablet | Mobile |
|--------|-------|-------|
| localhost_8888_enterprise-2_(Desktop) | localhost_8888_enterprise-2_(iPad) (3) | localhost_8888_enterprise-2_(Samsung Galaxy S20 Ultra) (6) |

### How to test the changes in this Pull Request:

1. Build the styles
2. Load `/enterprise`
3. Check layout and copy

### CODE REVIEW

File: source/wp-content/themes/wporg-main-2022/patterns/enterprise.php [hunk 1]
```diff
@@ -100,8 +94,8 @@
 <h3 class="wp-block-heading is-style-default has-inter-font-family has-large-font-size" style="font-style:normal;font-weight:600"><?php _e( 'Media and publishing', 'wporg' ); ?></h3>
 <!-- /wp:heading -->
 
-<!-- wp:paragraph {"style":{"spacing":{"padding":{"top":"var:preset|spacing|10"}}},"fontSize":"small"} -->
-<p class="has-small-font-size" style="padding-top:var(--wp--preset--spacing--10)"><?php _e( 'WordPress powers the sites people visit multiple times every day for news, information, and entertainment.', 'wporg' ); ?></p>
+<!-- wp:paragraph {"style":{"spacing":{"padding":{"top":"var:preset|spacing|10"}}}} -->
+<p style="padding-top:var(--wp--preset--spacing--10)"><?php _e( 'WordPress powers the sites people visit multiple times every day for news, information, and entertainment.', 'wporg' ); ?></p>
```
Comment (ryelle) [inline source/wp-content/themes/wporg-main-2022/patterns/enterprise.php]: We've got a block style for short text, I think it's meant for situations like this — it's used on all the little paragraphs on the homepage.

Here's the difference, it just tightens up the line height. Here each p on the left column has it, and the right is default.
Screenshot 2023-01-19 at 3 51 58 PM

**Review by ryelle — APPROVED** [review]
Quick code review looks good 👍🏻 

One minor comment more about the design, but feel free to take it or leave it.

File: source/wp-content/themes/wporg-main-2022/patterns/enterprise.php (hunk 1 above)
Comment (adamwoodnz) [reply source/wp-content/themes/wporg-main-2022/patterns/enterprise.php]: Ah yes I forgot about that, the design does have reduced line height, but it did originally have a smaller font too... anyway I think it's a good suggestion, thanks

Comment (adamwoodnz) [reply source/wp-content/themes/wporg-main-2022/patterns/enterprise.php]: Updated 🙏

**Pull request merged**